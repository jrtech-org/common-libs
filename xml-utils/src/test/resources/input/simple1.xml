<?xml version="1.0" encoding="UTF-8"?>
<Document xmlns="urn:test:xsd:aaachoice.001.001.01">
    <ChoiceTestDocument>
        <AcctId>
            <Prtry>
                <Id>jruac001</Id>
            </Prtry>
        </AcctId>
        <AcctNm></AcctNm>
        <OwnrId />
        <AcctSvcr />
        <OrdrOrgtrElgblty />
        <Order>
            <SubAcctForHldg>
                <Id>
                    <Prtry>
                        <Id>myac001</Id>
                    </Prtry>
                </Id>
            </SubAcctForHldg>
        </Order>
        <testNode>
            <Tp></Tp>
        </testNode>
    </ChoiceTestDocument>
</Document>
