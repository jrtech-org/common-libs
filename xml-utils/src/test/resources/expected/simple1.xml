<?xml version="1.0" encoding="UTF-8"?>
<Document xmlns="urn:test:xsd:aaachoice.001.001.01">
    <ChoiceTestDocument>
        <AcctId>
            <Prtry>
                <Id />
            </Prtry>
        </AcctId>
        <AcctNm />
        <AcctDsgnt />
        <OwnrId />
        <AcctSvcr />
        <OrdrOrgtrElgblty />
        <testNode>
            <Tp></Tp>
            <RcptId></RcptId>
        </testNode>
    </ChoiceTestDocument>
</Document>
